CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html)
 
+## [9.1.0] - Unreleased
```

Comment:
Unreleased should not have a version associated with it. It's OK for now, but just FYI

Reply:
Yeah, I just wanted to signify that this will be a separate major version. Next time I have extra time, I should look into changie. It'd simplify this a fair bit, I think.

Reply:
> Yeah, I just wanted to signify that this will be a separate major version

Major or minor version?

Reply:
Minor, all changes are backward compatible